test/index-spec.js

```diff
@@ -1,11 +1,13 @@
-const { start, stop } = require('../lib/index');
-const http = require('http');
-const path = require('path');
-const fs   = require('fs-extra');
-const nock = require('nock');
-const clientGenerator = require('bravado-client-generator');
+import { start, stop } from '../lib/index.js';
+import http from 'http';
+import path from 'path';
+import fs from 'fs-extra';
+import nock from 'nock';
+import * as url from 'url';
+import clientGenerator from 'bravado-client-generator';
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
 const testClientPath  = path.join(__dirname, 'testClient');
-require('should');
+import should from 'should'; // eslint-disable-line no-unused-vars
```

Comment:
is `should` still needed here? if so, does just `import 'should'` provide the same effect?

Reply:
yes yes it does. Good catch.